recipes/nolearn/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "nolearn" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "a7779c7d7d4bcf34d4db439df249eadbba04f3dd09ee7579f07ddf24e5bfc579" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy==1.10.4
+    - scipy==0.16.1
+    - scikit-learn==0.17
+    - joblib==0.9.3
+    - Theano==0.8
+    
+  run:
+    - python
+    - numpy==1.10.4
+    - scipy==0.16.1
+    - scikit-learn==0.17
+    - joblib==0.9.3
+    - Theano==0.8
+
+test:
+  imports:
+    - nolearn
+    - nolearn.tests
+
+about:
+  home: https://github.com/dnouri/nolearn
+  license: MIT
+  license_file: LICENSE.txt
+  summary: 'scikit-learn compatible neural network library for Python'
+
+  # The remaining entries in this section are optional, but recommended
+  description: |
+    nolearn is a Python package with abstractions around the library 
+    Lasagne <lasagne.readthedocs.org> and is compatible with scikit-learn.
+    nolearn supports Python 2.7 and Python 3.4.
+  doc_url: https://pythonhosted.org/nolearn/
+  dev_url: https://github.com/dnouri/nolearn
+
+extra:
+  recipe-maintainers:
+    - souravsingh
+    - dnouri
```

Comment:
@dnouri -- are you ok with being added as a co-maintainer of the `nolearn` recipe on conda-forge?